src/pkm/__test__/pk3.test.ts

```diff
@@ -1,130 +1,89 @@
-    tough: 255,
-    cute: 255,
-    sheen: 1,
-  }
-  blazikenOH.updateData(emeraldPKM)
-  expect(blazikenOH.contest).toStrictEqual({
-    cool: 30,
-    beauty: 255,
-    smart: 255,
-    tough: 255,
-    cute: 255,
-    sheen: 1,
-  })
-})
-
-test('gen 3 conversion to OHPKM and back is lossless', () => {
-  const ohPKM = new OHPKM(undefined, blazikenGen3)
-  // gaining cool contest points
-  const gen3PKM = new PK3(undefined, undefined, ohPKM)
-  expect(blazikenGen3.bytes).toEqual(gen3PKM.bytes)
-})
-
-test('pk3 and ohpkm have the same gen345Lookup key', () => {
-  const ohPKM = new OHPKM(undefined, blazikenGen3)
-  expect(getMonGen345Identifier(ohPKM)).toEqual(getMonGen345Identifier(blazikenGen3))
-})
-
-test('gen 6+ nickname accuracy', () => {
-  const converted = new PK3(undefined, undefined, slowpokeOH)
-  expect(converted.nickname).toBe(slowpokeOH.nickname)
-})
-
-test('gen 6+ shiny accuracy', () => {
-  const converted = new PK3(undefined, undefined, slowpokeOH)
-  if (!slowpokeOH.personalityValue) {
-    throw Error('mon has no personality value')
-  }
-  expect(converted.isShiny).toBe(slowpokeOH.isShiny)
-})
-
-test('gen 6+ nature accuracy', () => {
-  const converted = new PK3(undefined, undefined, slowpokeOH)
-  expect(converted.nature).toBe(slowpokeOH.nature)
-})
+import PK3 from '../PK3'; // Adjust the import path as needed
+// import { Ball, Gen3ContestRibbons, Gen3StandardRibbons } from 'pokemon-resources';
+// import { NationalDex } from 'pokemon-species-data';
+// import * as types from '../util/types'; // Adjust based on actual path
+
+describe('PK3 Class', () => {
+  const buffer = new Uint8Array([
+    0x0C, 0xEF, 0x02, 0xD2, 0x6B, 0x99, 0x37, 0x75, 0xBB, 0xE0, 0xD5, 0xDF, 0xD5, 0xEE, 0xD5, 0xE6,
+    0xFF, 0x00, 0x02, 0x02, 0xBE, 0xDD, 0xD5, 0xE1, 0xE3, 0xE2, 0xD8, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x41, 0x00, 0x00, 0x00, 0xC2, 0x70, 0x07, 0x00, 0x00, 0x46, 0x00, 0x00, 0x5E, 0x00, 0xF7, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x0A, 0x0F, 0x00, 0x00, 0x05, 0x00, 0x00, 0xFC, 0xFC, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x12, 0x05, 0x26, 0xEF, 0x97, 0xEB, 0x81, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xBC, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00
+  ]);
+
+//   const buffer = new Uint8Array([
+//     0xF6, 0x1C, 0x63, 0xC8, 0x05, 0x46, 0xD7, 0x32, 0xCD, 0xDD, 0xD6, 0xD9, 0xE6, 0xDD, 0xE9, 0xE7, 
+//     0xFF, 0xFF, 0x02, 0x02, 0xCC, 0xD9, 0xD8, 0xFF, 0xD5, 0xE0, 0xD8, 0x00, 0x3C, 0x54, 0x00, 0x00, 
+//     0xF8, 0x00, 0xC8, 0x00, 0xD0, 0x12, 0x13, 0x00, 0x00, 0x2A, 0x00, 0x00, 0xF2, 0x00, 0x59, 0x00, 
+//     0x5D, 0x01, 0x9D, 0x00, 0x0F, 0x0A, 0x14, 0x0A, 0x04, 0xFC, 0x01, 0xFC, 0x01, 0x00, 0xFF, 0xFF, 
+//     0xFF, 0xFF, 0xFF, 0xFF, 0x00, 0x5B, 0x00, 0x22, 0xFF, 0xFF, 0xFF, 0x3F, 0x24, 0x49, 0x0C, 0x00, 
+//     0x00, 0x00, 0x00, 0x00, 0x64, 0x00, 0x56, 0x01, 0x56, 0x01, 0x93, 0x01, 0x00, 0x01, 0xDD, 0x00, 
+//     0xCB, 0x00, 0xEC, 0x00, 
```

Comment:
These commented lines should be removed too

Reply:
Happy to remove the imports. but the commented out `ByteArray` represents another pokemon you could test. There's probably a fancier way of swapping between the pokemon however.

Reply:
Is this better I renamed the second buffer array